### PR TITLE
keyboard navigation

### PR DESCRIPTION
**Pull request by KittenHero**

### CODE REVIEW

File: src/components/grids/PlanGrid.tsx [hunk 1]
```diff
@@ -178,9 +179,89 @@ const PlanGrid = () => {
     setLayout(newLayout);
   };
 
+  const handleMove = (dx: number, dy: number) => {
```
Comment (plateupplanner) [inline src/components/grids/PlanGrid.tsx]: Could you change this to `handleItemMove`, and change `handleShiftSelect` to `handleSelectionMove`?

File: src/components/modals/infoModal/styled.ts [hunk 2]
```diff
@@ -4,4 +4,23 @@ export const InfoModal = styled.div`
   p {
     margin-bottom: 8px;
   }
+  kbd {
```
Comment (plateupplanner) [inline src/components/modals/infoModal/styled.ts:7]: Can't tell what `kbd` stands for, maybe rename to `keybind`? (I love how it shows up btw!)

File: src/components/menu/Menu.tsx [hunk 3]
```diff
@@ -45,22 +57,104 @@ const Menu = ({ showMenu = true, mode }: Props) => {
...
+  const handleUnfocusSearch = () => {
+    if (itemFocus !== null) {
+      itemRefs.current[itemFocus]?.blur();
+    }
+    setShowMenu?.(false);
+    setItemFocus(null);
+    searchRef.current?.blur();
+  };
+
+  const handleItemNavigation = (
```
Comment (plateupplanner) [inline src/components/menu/Menu.tsx]: Rename to `handleMenuNavigation`

**Review by plateupplanner — APPROVED** [review]

File: src/components/modals/infoModal/styled.ts (hunk 2 above)
Comment (KittenHero) [reply src/components/modals/infoModal/styled.ts:7]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd

**Pull request merged**